BlazarUI/app/scripts/pages/app.jsx

```diff
@@ -4,12 +4,15 @@ import BuildsSidebarContainer from '../components/sidebar/BuildsSidebarContainer
 import Modal from 'react-bootstrap/lib/Modal';
 import Button from 'react-bootstrap/lib/Button';
 import Input from 'react-bootstrap/lib/Input';
+import {bindAll} from 'underscore';
 
 class App extends Component {
 
   constructor() {
+    bindAll(this, 'collapseSidebar');
     this.state = {
-      showModal: (!window.config.apiRoot)
+      showModal: (!window.config.apiRoot),
+      sidebarCollapsed: localStorage.sidebarCollapsed === 'true' ? true : false
```

Comment:
thanks @gusvargas - I noticed these as well and was planning on reviewing with kevin